Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
```

Comment:
Each command can be clearer with better comments. Consider adding a comment above each task explaining its purpose and any important details (e.g., any dependencies required, environment variables needed).